# raw accel data used for logging, published at the highest rate possible if enabled

float32 x_raw
float32 y_raw
float32 z_raw
